lib/aruba/platform.rb

```diff
@@ -230,6 +228,16 @@ def which(program, path = ENV['PATH'])
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/CyclomaticComplexity
 
+    # return true if the OS running is a Unix system according to FFI
+    def unix?
+      FFI::Platform.unix?
+    end
```

Comment:
Unix will be default. So we do not need this method. All class handling unix will return true on `match?`. Otherwise we would need to check on `solaris?`, `mac?` etc. as well.